EvidenceApi.Tests/V1/E2ETests/DocumentSubmissionsTest.cs

```diff
@@ -22,171 +22,27 @@ public class DocumentSubmissionsTest : IntegrationTests<Startup>
         private readonly IFixture _fixture = new Fixture();
         private Claim _createdClaim;
         private Document _document;
-        private S3UploadPolicy _createdUploadPolicy;
-        private Guid id = Guid.NewGuid();
+        private readonly Guid _id = Guid.NewGuid();
 
         [SetUp]
         public void SetUp()
         {
             _document = _fixture.Build<Document>()
-                .With(x => x.Id, id)
+                .With(x => x.Id, _id)
                 .Create();
             _createdClaim = _fixture.Build<Claim>()
                 .With(x => x.Document, _document)
                 .Create();
 
-            _createdUploadPolicy = _fixture.Create<S3UploadPolicy>();
-
-            DocumentsApiServer.Given(
-                Request.Create().WithPath("/api/v1/claims")
-            ).RespondWith(
-                Response.Create().WithStatusCode(201).WithBody(
-                    JsonConvert.SerializeObject(_createdClaim)
-                )
-            );
-
             DocumentsApiServer.Given(
                 Request.Create().WithPath($"/api/v1/claims/{_createdClaim.Id}").UsingGet()
             ).RespondWith(
                 Response.Create().WithStatusCode(200).WithBody(
                     JsonConvert.SerializeObject(_createdClaim)
                 )
             );
-
-            DocumentsApiServer.Given(
-                Request.Create().WithPath($"/api/v1/documents/{id}")
-            ).RespondWith(
-                Response.Create().WithStatusCode(200)
-            );
-
-            DocumentsApiServer.Given(
-                Request.Create().WithPath($"/api/v1/documents/{id}/upload_policies")
-            ).RespondWith(
-                Response.Create().WithStatusCode(200).WithBody(
-                    JsonConvert.SerializeObject(_createdUploadPolicy)
-                )
-            );
-        }
-
-
-        [Test]
-        public async Task CanCreateDocumentSubmissionWithValidParams()
```

Comment:
Moved the tests from this file to `EvidenceResquestTests` because the endpoint is in EvidenceRequest, not DocumentSubmission